cts/support/fence_dummy.in

```diff
@@ -500,6 +509,11 @@ def main():
         else:
             exitcode = 1
 
+    elif action == "off":
+        if "-F" in options:
+            time.sleep(int(options["-F"]))
```

Comment:
this will be in addition to any delay from -f -- just making sure that was intentional

Reply:
Yep that was intentional and on top the easiest least intrusive.
For my purpose I needed everything to finish within timeouts but the actual fencing.
So it is assumed that -f isn't used in combination.

Reply:
stated that it is an additional wait in description.